docs/src/app/components/Divider/README.md

```diff
@@ -0,0 +1,5 @@
+## Divider
+
+<a href="https://www.google.com/design/spec/components/dividers.html" target="_blank">Dividers</a> group and separate content within lists and page layouts. The divider is a thin rule, lightweight yet sufficient to distinguish content visually and spatially.
```

Comment:
Yeah, I initally made a normal link, but I thought it'd be better to open the spec in a window/tab.
Is there a way to do that with MD?
If not, I can change anyway...